ooniprobe/View/Informed consent/PopQuizViewController.m

```diff
@@ -0,0 +1,108 @@
+#import "PopQuizViewController.h"
+
+@interface PopQuizViewController ()
+
+@end
+
+@implementation PopQuizViewController
+@synthesize question_number;
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    [self.trueButton addTarget:self action:@selector(answer:) forControlEvents:UIControlEventTouchUpInside];
+    [self.trueButton setTitle:[NSLocalizedString(@"_true", nil) uppercaseString] forState:UIControlStateNormal];
+    
+    [self.falseButton addTarget:self action:@selector(answer:) forControlEvents:UIControlEventTouchUpInside];
+    [self.falseButton setTitle:[NSLocalizedString(@"_false", nil) uppercaseString] forState:UIControlStateNormal];
+    
+    [self.closeView addTarget:self action:@selector(dismissPopup) forControlEvents:UIControlEventTouchUpInside];
+    
+    self.cointainerWindow.layer.cornerRadius = 12;
+    self.cointainerWindow.layer.masksToBounds = true;
+    
+    [self.titleLabel setText:NSLocalizedString(@"pop_quiz", nil)];
+    [self reloadQuestion];
+}
+
+- (void)didReceiveMemoryWarning {
+    [super didReceiveMemoryWarning];
+    // Dispose of any resources that can be recreated.
+}
+
+- (void)reloadQuestion{
+    NSMutableAttributedString *question_intro = [[NSMutableAttributedString alloc] initWithString:[NSString stringWithFormat:@"%@ %ld/2\n\n", NSLocalizedString(@"question", nil), question_number]];
+    [question_intro addAttribute:NSFontAttributeName
+                           value:[UIFont fontWithName:@"FiraSans-Regular" size:17]
+                           range:NSMakeRange(0, question_intro.length)];
+    NSMutableAttributedString *question_text;
+    if (question_number == 1)
+        question_text = [[NSMutableAttributedString alloc] initWithString:NSLocalizedString(@"question_1", nil)];
+    else if (question_number == 2)
+        question_text = [[NSMutableAttributedString alloc] initWithString:NSLocalizedString(@"question_2", nil)];
```

Comment:
In general, I would make sure that, when the impossible happens, we `abort()`.

Reply:
We don't have to abort anything here. It's a label.